### PR TITLE
Implement resistor-colors exercise

### LINKED ISSUE #641: Curriculum: add new exercise resistor-colors (round 2)
- [ ] add new exercise `resistor-colors` (Also known as Resistor Color II, follow-up for #640)
- [ ] as an experiment 🔬 add this as core right after `resistor-color`

After a while, check the metrics and see if we should:
- keep `resistor-colors` as core
- keep `resistor-color` as core
- keep both as core

### PR DESCRIPTION
**Pull request by tejasbubane**
Problem spec PR: https://github.com/exercism/problem-specifications/pull/1466

Closes #641

### CODE REVIEW

**Comment by SleeplessByte** [comment]
Looks fine on first skim. 

Don't merge ⚠️ yet -- Maud and I are figuring out if we want to combine #639 #640 #641 in one go.

File: config.json [hunk 1]
```diff
@@ -1308,7 +1308,17 @@
         "math"
       ]
     },
-
+    {
+      "slug": "resistor-colors",
+      "uuid": "de800041-3dcc-41b9-b101-7314ff685c93",
+      "core": false,
+      "unlocked_by": "pangram",
```
Comment (SleeplessByte) [inline config.json]: Should be unlocked_by: "resistor-color" (#640) as per the track-tool.

```suggestion
      "unlocked_by": "resistor-color",
```

**Comment by SleeplessByte** [comment]
Discussed this with Maud. We'll be merging #640 + #639  + #641 simultaneously as a track anatomy experiment.

File: config.json (hunk 1 above)
Comment (tejasbubane) [reply config.json]: We don't have `resistor-color`. Might need to implement that first.

Comment (tejasbubane) [reply config.json]: Ohh ok thats what #640 is for.

Comment (tejasbubane) [reply config.json]: Also need to make this core as per #641

Comment (SleeplessByte) [reply config.json]: You got it!

Comment (SleeplessByte) [reply config.json]: #640 is in #648

Comment (SleeplessByte) [reply config.json]: @tejasbubane want to move this up in `config.json` and make it `core`? If you don't have time I can take care of it.

**Comment by SleeplessByte** [comment]
Don't forget to update and copy
- `package.json`
- `babel.config.js`
- `.eslintrc`

**Comment by F3PiX** [comment]
Fantastic!

**Comment by tejasbubane** [comment]
@SleeplessByte Done

**Comment by F3PiX** [comment]
Yay!

**Pull request merged**